tests/test_readers.py

```diff
@@ -179,6 +180,18 @@ def checkExposureFitsReader(self, exposureIn, fileName, dtypesOut):
         reader = ExposureFitsReader(fileName)
         self.assertIn('EXPINFO_V', reader.readMetadata().toDict(), "metadata is automatically versioned")
         reader.readMetadata().remove('EXPINFO_V')
+        # ensure EXTNAMEs can be read and make sense
+        astropyReadFile = astropy.io.fits.open(fileName)
+        extnames = ('IMAGE', 'MASK', 'VARIANCE', 'ARCHIVE_INDEX',
```

Comment:
Please use a `set` rather than `tuple` -- it's slightly more efficient for the `in` operator.